node/src/components/storage.rs

```diff
@@ -1669,16 +1669,24 @@ fn garbage_collect_block_body_v2_db(
 ) -> Result<(), LmdbExtError> {
     // This will store all the keys that are reachable from a block header, in all the parts
     // databases (we're basically doing a mark-and-sweep below).
-    let mut live_digests = HashSet::new();
+    // The entries correspond to: the block_body_v2_db, deploy_hashes_db, transfer_hashes_db,
+    // proposer_db respectively.
+    let mut live_digests = [
```

Comment:
Maybe `live_digests[0]` should be its own variable, because it's treated differently and `live_digests_index` is never 0?

Reply:
It's not in this loop, but it's used the same as the other indices later.